### ISSUE TITLE
Adding Mission Area

### ISSUE THREAD

**Issue by Keristiena** [uno-isqa-8950/uno-cpi#516]
Ensure that the Admin has the ability to add a mission area and can change the designated primary mission area of existing community partners/add new community partners with this mission area. 
This should work and be reflected in all reports/charts/maps (plus legends)

**Comment by caadler** [comment]
Splitting into multiple tickets/sprints.  Admin DB add will be worked on first, then Reports/Charts second.

**Comment by jnabity** [comment]
Need to look specifically at this updating and reflecting properly for maps.

**Comment by jnabity** [comment]
Blocked by #830

**Comment by EdemD** [comment]
@jnabity please go ahead and test the Legislative district map. Per @vbhat0891: "I updated the color schema for legislative district map. @Craig shared a Power BI specific color schema and the legislative district map can have additional mission areas added. It is ticket #516 which apparently says blocked by GeoJSON scheduler and it is not. Kindly go ahead and test. I will extend it to other maps once it is tested."
Just fyi. Thanks

**Comment by jnabity** [comment]
Tested legislative district. Test cases below:
https://docs.google.com/spreadsheets/d/1JY7VRaLshlpIpYqF1Pas53hTunUw7wqHse1GYDp2vDY/edit#gid=227356338
Are we using this ticket to track and test the other maps as well?
FYI - @EdemD  @jerrodanzalone  @vbhat0891

**Comment by EdemD** [comment]
@jnabity this change should reflect on all of the maps. I don't believe there is another ticket for the other maps. I'll say keep the ticket in Review/QA for now until @vbhat0891 make the changes to two other maps(Community Partner and Community Partner Type map) and push soon. Thanks

**Comment by jnabity** [comment]
Tested this for all applicable maps. Test script below:
https://docs.google.com/spreadsheets/d/1JY7VRaLshlpIpYqF1Pas53hTunUw7wqHse1GYDp2vDY/edit#gid=272336465
